virtualizor.py

```diff
@@ -105,17 +105,13 @@ def __init__(self, conf):
         self.conn = libvirt.open('qemu+ssh://root@%s/system' %
                                  conf.target_host)
 
-        try:
-            self.public_net = self.conn.networkLookupByName(
-                conf.public_network)
-        except libvirt.libvirtError:
-            logging.error("public_network %s should be active and have "
-                          "internal DHCP turned on." % (conf.public_network))
-            raise self.MissingPublicNetwork()
-
     def create_networks(self, conf, install_server_info):
         net_definitions = {
-            ("%s_sps" % conf.prefix): {}
+            ("%s_sps" % conf.prefix): {},
+            ("%s" % conf.public_network): 
+                {"dhcp":{"address":"10.0.0.1","netmask":"255.255.255.0",
```

Comment:
I think a 192.168.140.0/24 is more likely to be unused. It's a bit common to see the full 10/8 used.